src/pages/derivx/_start-derivx.tsx

```diff
@@ -192,7 +209,7 @@ const StyledText = styled(Text)`
     }
 `
 
-const StartDerivX = () => {
+const StartDerivX: React.FC<StartDerivXType> = () => {
```

Comment:
```suggestion
const StartDerivX: ReactElement = () => {
```

Reply:
My fail. Now I have understanding what and how need to do :) Thanks a lot